tests/infra/e2e_args.py

```diff
@@ -218,7 +218,7 @@ def cli_args(add=lambda x: None, parser=None, accept_unknown=False):
         "--join-timer-s",
```

Comment:
This is to avoid marking a backup as "timed out" when it's been trusted but hasn't yet polled the primary again to join the consensus. This was set to a high value here historically, because the quote verification was expensive on the primary. I believe that recent changes to the DCAP client means that we cache quotes + collaterals a lot more efficiently now so this shouldn't be an issue anymore.